src/sqswatcher/sqswatcher.py

```diff
@@ -263,11 +263,16 @@ def _parse_sqs_messages(messages, table):
     return update_events.values()
 
 
-def _process_compute_ready_event(message_attrs, message):
+def _process_compute_ready_event(sqs_config, message_attrs, message):
```

Comment:
let's only pass the required parameters such as region and proxy_config.

As a general comment we should really get rid of region and proxy_config and set them as env variable right after we read the config. No need to do it now since unrelated.